src/main.ts

```diff
@@ -12,17 +12,24 @@ type StringOrMatchConfig = string | MatchConfig;
 
 async function run() {
   try {
-    const token = core.getInput('repo-token', {required: true});
-    const configPath = core.getInput('configuration-path', {required: true});
+    const token = core.getInput("repo-token", { required: true });
+    const configPath = core.getInput("configuration-path", { required: true });
+    const syncLabels = !!core.getInput("sync-labels", { required: false });
```

Comment:
sync-labels still triggering when set to `false` (bug #104), or when unset.

I think the bug is on this line - getInput returns a string, and `!!"false"` is true, just like `!!"true"`.

Reply:
Doh! When I orginally wrote this code, I don't think I tested setting the input to `false`. You can just leave the input out to disable syncing as a workaround until this bug is fixed.